doc/source/merging.rst

```diff
@@ -551,6 +552,18 @@ standard database join operations between DataFrame objects:
 
   .. versionadded:: 0.17.0
 
+- ``validate`` : {None, '1:1', '1:m', 'm:1', 'm:m', "one_to_one", "one_to_many", "many_to_one", "many_to_many"}, default None
```

Comment:
I would split this up and put the choices on the 2nd line, otherwise this gets too long (or you can use ``\`` to split the line). maybe check how this all renders.